packages/frontend/src/features/catalog/components/CatalogTableListItem.tsx

```diff
@@ -53,68 +55,68 @@ export const CatalogTableListItem: FC<React.PropsWithChildren<Props>> = ({
             <Group
                 noWrap
                 position="apart"
-                py="sm"
+                spacing="xs"
                 px="xs"
                 sx={(theme) => ({
-                    minHeight: 40,
-                    borderRadius: theme.radius.sm,
-                    padding: theme.spacing.md,
+                    minHeight: 48,
+                    borderBottom: isLast
+                        ? 'none'
+                        : `1px solid ${theme.colors.gray[2]}`,
                     backgroundColor: hovered
-                        ? theme.colors.gray[2]
-                        : theme.colors.gray[1],
+                        ? theme.colors.gray[1]
+                        : 'transparent',
                     border: isSelected
                         ? `2px solid ${theme.colors.blue[6]}`
                         : undefined,
                     cursor: 'pointer',
+                    borderTopLeftRadius: isFirst ? theme.radius.lg : 0,
```

Comment:
Could you do this with css or do you need the control these props get you?

Like:
```
'&:first-of-type': {
    borderTopLeftRadius: theme.radius.xl,
},
```

If you jsut like props better, that's fine too :D

Reply:
It works well for the `first-of-type` but doesn't work for the `last-of-type` because some siblings are also `div`s that aren't a `CatalogTableListItem`.Group. I would have to override the type of this element to achieve the same with this selector... I could also apply the styles on the parent, but I think that would make it less easier to read/track. Hope it's ok 👍